src/ui/layout/search/GrandSearchSpec.js

```diff
@@ -29,7 +29,7 @@ import GrandSearch from './GrandSearch.vue';
 import ExampleTagsPlugin from '../../../../example/exampleTags/plugin';
 import DisplayLayoutPlugin from '../../../plugins/displayLayout/plugin';
 
-xdescribe("GrandSearch", () => {
+describe("GrandSearch", () => {
```

Comment:
Last week we disabled pretty much any test that touches the router since it was causing random test flakes. GrandSearch had a few of these so it was on the chopping block. I'm seeing that this test added is already failing due to the async timeout issue. I think there is some logic triggered by a router path change that's not being handled?

Either way, I don't think this should block this PR. Maybe we should try adding e2e coverage for this specific issue instead and keep the search tests skipped for now.

Reply:
Might be the latest merge from master? I'll check, and if I can't reproduce locally, disable and try for an e2e test.